pyrasterframes/src/main/python/docs/aggregation.pymd

```diff
@@ -10,13 +10,65 @@ import os
 spark = create_rf_spark_session()
 ```
 
-## Cell Counts & Tile Mean
+There are 3 types of aggregate functions: tile aggregate, DataFrame aggregate, and element-wise local aggregate. In the @ref:[`tile aggregate functions`](reference.md#tile-statistics), we are computing a statistical summary per row of a `tile` column in a DataFrame. In the @ref:[`DataFrame aggregate functions`](reference.md#aggregate-tile-statistics), we are computing statistical summaries over all of the cell values *and* across all of the rows in the DataFrame or group. In the  @ref:[`element-wise local aggregate functions`](reference.md#tile-local-aggregate-statistics), we are computing the element-wise statistical summary across a DataFrame or group of tiles.
+
+## Tile Mean
+
+We can illustrate this difference in computing tile mean. In the first code block we are using @ref:[`rf_tile_mean`](reference.md#rf-tile-mean) to computer the tile aggregate mean of cells in each row of column `tile`. The mean of each tile is computed separately, so the first mean is 1.0 and the second mean is 3.0.
+
+```python
+import pyspark.sql.functions as F
+
+spark.sql("""
+SELECT 1 as id, rf_make_ones_tile(5, 5, 'float32') as t
+UNION
+SELECT 2 as id, rf_local_multiply(rf_make_ones_tile(5, 5, 'float32'), 3.0) as t
+""").select(F.col('id'), rf_tile_mean(F.col('t'))).show(10, False)
+```
+
+In this next code block we are using @ref:[`rf_agg_mean`](reference.md#rf-agg-mean) to compute the DataFrame aggregate, which averages 25 values of 1.0 and 25 values of 3.0, across the fifty cells in two rows. Note that only a single row is returned since the average is computed over the full DataFrame.
+
+```python
+import pyspark.sql.functions as F
+
+spark.sql("""
```

Comment:
since we are constucting the same df I'd recommend at line 22 assigning the spark.sql result to a variable.